airflow-core/src/airflow/ui/src/i18n/locales/de/common.json

```diff
@@ -0,0 +1,78 @@
+{
+  "defaultToGraphView": "Graph-Ansicht als Standard",
+  "defaultToGridView": "Gitter-Ansicht als Standard",
+  "logout": "Abmelden",
+  "switchToDarkMode": "Zum Dunkelmodus wechseln",
+  "switchToLightMode": "Zum Hellmodus wechseln",
+  "timezone": "Zeitzone",
+  "user": "Benutzer",
+  "language": {
+    "chinese": "Traditionelles Chinesisch",
+    "english": "Englisch",
+    "german": "Deutsch",
+    "select": "Sprache wählen"
+  },
+  "nav": {
+    "home": "Start",
+    "assets": "Datenbestände",
+    "browse": "Browsen",
+    "admin": "Verwaltung",
+    "docs": "Dokumentation",
+    "plugins": "Plug-ins"
+  },
+  "browse": {
+    "auditLog": "Prüf-Log",
+    "xcoms": "Datenwerte"
```

Comment:
hmm maybe something that includes communication could be more clear? I am a bit worried if "Daten" is used it could be too connected to "Datenbestände" for newer users, despite xcoms not being related to assets. 

"Task Kommunikation" ? 

(nur ein Denkanstoss, ich bin mir selbst nicht sicher, was mehr Sinn macht hier 😅 )

Reply:
Okay, taking "Task Kommunikation (XCom)" now. This then is clear and it references the original term